packages/config/config/devices/0x0063/templates/config_template.json

```diff
@@ -0,0 +1,468 @@
+		"$import": "~/templates/master_template.json#dimming_timing"
+	},
+	"dim_threshold_min": {
+		"label": "Minimum Dim Threshold",
+		"description": "Set the minimum dimmer threshold when manually or remotely controlled",
+		"$import": "~/templates/master_template.json#base_1-99_nounit"
+	},
+	"dim_threshold_max": {
+		"label": "Maximum Brightness Threshold",
+		"description": "Set the maximum brightness threshold when manually or remotely controlled",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 99,
+		"defaultValue": 99,
+		"unsigned": true
+	},
+	"default_brightness_level": {
+		"label": "Default Brightness Level",
+		"description": "Set the default brightness level that the dimmer will turn on when being controlled manually",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 99,
+		"defaultValue": 0,
+		"unsigned": true
+	},
+	"dim_level": {
+		"label": "Dim level",
+		"description": "Change brightness of associated light bulb(s)",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"defaultValue": 255,
+		"unsigned": true,
+		"options": [
+			{
+				"label": "Default",
+				"value": 255
+			}
+		]
+	},
+	"switch_level": {
+		"label": "Switch Level",
+		"description": "Set power level for plain switch functionality",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 99,
+		"defaultValue": 0
+	},
+	"dim_rate": {
+		"label": "Dim Rate",
+		"description": "Adjust the speed at which the ramps to a specific value other than 0 and FF",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Dim Quickly",
+				"value": 0
+			},
+			{
+				"label": "Dim Slowly",
+				"value": 1
+			}
+		]
+	},
+	"timeout_duration": {
+		"label": "Timeout Duration",
+		"description": "Amount of time the light stays on after motion stops",
+		"valueSize": 1,
+		"defaultValue": 5,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Test (5s)",
+				"value": 0
+			},
+			{
+				"label": "1 minute",
+				"value": 1
+			},
+			{
+				"label": "5 minutes",
+				"value": 5
+			},
+			{
+				"label": "15 minutes",
+				"value": 15
+			},
+			{
+				"label": "30 minutes",
+				"value": 30
+			},
+			{
+				"label": "Disable timeout",
+				"value": 255
+			}
+		]
+	},
+	"pir_sensitivity": {
+		"label": "PIR Sensitivity",
+		"valueSize": 1,
+		"defaultValue": 3,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Low",
+				"value": 1
+			},
+			{
+				"label": "Medium",
+				"value": 2
+			},
+			{
+				"label": "High",
+				"value": 3
+			}
+		]
+	},
+	"motion_sensitivity": {
+		"label": "Motion Sensor Sensitivity",
+		"valueSize": 1,
+		"defaultValue": 2,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "High",
+				"value": 1
+			},
+			{
+				"label": "Medium",
+				"value": 2
+			},
+			{
+				"label": "Low",
+				"value": 3
+			}
+		]
+	},
```

Comment:
These are the same aside from the default value, aren't they?

Reply:
I looked at this, unfortunately not. Look at the options, one counts down and one counts up.